Python/ceval.c

```diff
@@ -6875,12 +6875,15 @@ void
 PyThreadState_EnterTracing(PyThreadState *tstate)
 {
     tstate->tracing++;
+    tstate->cframe->use_tracing = 0;
```

Comment:
Out of curiosity, under what conditions would it not already be 0?

Reply:
When running normal code under tracing.
`use_tracing` is set when we are tracing, this function is called when we enter the tracing function.